<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrayDeleteWidget</class>
 <widget class="QWidget" name="ArrayDeleteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ArrayValueDeleteDrawingWidget" name="widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="DeleteAt0">
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除索引０位置元素</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteAt2">
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除索引2位置元素</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteAt4">
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除索引４位置元素</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>数组在指定位置删除元素时，
将指定位置元素移出数组，
从删除元素下一元素开始，
到数组尾元素为止，
依次前移一个单位．
删除后，数组有效元素个数减少了１．</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ArrayValueDeleteDrawingWidget</class>
   <extends>QWidget</extends>
   <header>Ui/Drawing/arrayvaluedeletedrawingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
